### ISSUE TITLE
Double Macaroon input in LND service settings

### ISSUE THREAD

**Issue by hubiktomas** [btcpayserver/btcpayserver#473]
In the Server Settings > Services > lnd-grpc/lnd-rest > More Details I can see two inputs - Macaroon and AdminMacaroon with the same content - both contain Admin Macaroon. It is an unnecessary duplicity. I would call it just an Admin Macaroon so it is obvious, what it is.

**Comment by bitcoinshirt** [comment]
Replicated. 
Admin macaroon and Macaroon fields have the same value.
![2018-12-28_11-30-52](https://user-images.githubusercontent.com/36959754/50512773-2e50de00-0a94-11e9-8148-d760dc320d96.png)

ping @NicolasDorier

**Comment by NicolasDorier** [comment]
Yes I agree, will do

**Comment by NicolasDorier** [comment]
Ok so I think I will keep the `Macaroon` entry motivation:

1. Many videos has been done before we exposed all the macaroons, people following the video would not know which macaroon to take.
2. A noob can get confused about which macaroon to choose for what he is doing. Given all the choices, `Macaroon` seems the safest, and because it is the `AdminMacaroon` underneath, things will just work for him.

toughts?

**Comment by bitcoinshirt** [comment]
Macaroon entry makes more sense to me as well.

**Comment by bitcoinshirt** [comment]
It appears I misunderstood your post @NicolasDorier.
You would like to leave both as it is, because we already have sufficient amount of videos and documentation explaining what to add, is that correct?

I think if you just have macaroon field and remove admin macaroon the effect will be the same. No need to duplicate stuff, it's confusing.

**Comment by hubiktomas** [comment]
I am also not really happy about the duplication. And I am not sure if the argument of having videos is good to teach people something that does not exist. LND has three macaroon types and these types should be clearly named and displayed. We can also add some explanation about what each macaroon does for people who do not know, which one to choose. I think that having duplicate entries can be confusing as @bitcoinshirt said and having non-standard names  can be even worse for people who know LND and have not seen our videos.

**Comment by hubiktomas** [comment]
Hmm, that is true. The scenario without duplicate could be solved by adding appropriate description above the macaroons section, but I agree, that for users not familiar with what they are doing, Lightning network may still cause problems. In case without the duplicate, they may just not know, which macaroon to use and on third attempt they may succeed. With the duplicate, they may sacrifice their security by using too strong macaroon as a first attempt as it is called just "Macaroon". I would still prefer scenario with scratching head than scenario that is designed to sacrifice security by default. But I get your point.

**Comment by NicolasDorier** [comment]
Problem is that a malicious wallet will just tell the clueless user "Give me the admin Macaroon"

The user who understand things though is tech savvy enough to pick the proper one and not the default one.

**Comment by NicolasDorier** [comment]
I propose we close this. I don't think removing the macaroon stuff. There is no good UI that won't give headache and user question.

**Comment by pavlenex** [comment]
Agreed, @hubiktomas what are your thoughts?
We haven't received a single complaint so far about double fields, nor anyone messed things up. Close, and re-open if people report they're confused?

**Comment by hubiktomas** [comment]
It is not an issue on my side as I am using my instance just myself, not hosting for others. So I know, what it is and I also know, how LND macaroons work. I just found confusing that there are two fields with the same content and different name while one of them officially does not exist. I also see potential security issue here. That is why I reported this as a bug. If it is intentional not to break some old tutorials that refer to "macaroon" field, I can live with that even though I do not like it. :)
